src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNICommon.cs

```diff
@@ -244,7 +325,7 @@ internal static IPAddress[] GetDnsIpAddresses(string serverName)
             using (TrySNIEventScope.Create(nameof(GetDnsIpAddresses)))
             {
                 SqlClientEventSource.Log.TrySNITraceEvent(nameof(SNICommon), EventType.INFO, "Getting DNS host entries for serverName {0}.", args0: serverName);
-                return Dns.GetHostAddresses(serverName);
+                throw ADP.SSLCertificateAuthenticationException(Strings.SQL_RemoteCertificateNotAvailable);
```

Comment:
Looks like this was accidentally put on the wrong line.

Reply:
oops, this is GitHub Copilot insertion. I did not pay attention to this...